src/dataproc/generate_data.py

```diff
@@ -131,7 +132,7 @@ def generate_config(config_df, data_df, mode: str):
     ]
 
     # The value to use for type, based on the heuristic index above
-    values = ["stack", "pct", "ten", "hours", None]
+    values = ["stack", "pct", "pct", "ten", "hours", None]
```

Comment:
Just to be sure, `"pct"` is supposed to be listed twice?

Reply:
Yep! It's because we're treating the gender question as a 'special exception percent'.